coredns/on-premise-dns-zone/README.md

```diff
@@ -1,80 +1,80 @@
-
-2. Select 'Configure' next to the CoreDNS integration in the domain settings.
-
-3. Leave the Cluster Identifier value as `default`. 
+1. From the DNSimple application, navigate to the 'Domain->Integrations' page and select 'Configure' next to the CoreDNS integration.
+![domain_integrations](./images/domain_integrations.png)
 
-4. **Note** down your DNSimple `account_id` in the example Corefile.
+2. Leave the Cluster Identifier value as `default` and note down your DNSimple `account_id` from the provided Corefile.
+![coredns_setup](./images/coredns_setup.png)
 
-5. Click 'Create Integration' and verify the CoreDNS integration is active.
+3. Click 'Create Integration' and verify the CoreDNS integration is active.
+![coredns_active](./images/coredns_active.png)
 
-### Step 2: Compile/Install CoreDNS
+### Step 2: Compile or Install CoreDNS
 
 <details>
-<summary>Option A: Downloading the Precompiled Binary</summary>
+<summary>A: Compile from Source</summary>
 <br>
-1. Go to the [DNSimple CoreDNS releases page](https://github.com/dnsimple/coredns-dnsimple/releases)
-2. Download the latest release for your OS
-3. Extract and move the binary to a location in your PATH
-</details>
 
-<details>
-<summary>Option B: Build from Source</summary>
-<br>
-1. Clone the CoreDNS repository
-Clone the CoreDNS source code from its official GitHub repository.
+1. Clone the CoreDNS source code from its official GitHub repository:
 
 ```shell
 git clone https://github.com/coredns/coredns.git
 ```
 
-2. Navigate to the CoreDNS directory
-Switch to the coredns directory which has been cloned to your local system.
+2. Navigate to the CoreDNS directory:
 
 ```shell
 cd coredns
 ```
 
-3: Add the DNSimple Plugin to `plugin.cfg`
-Modify the plugin.cfg file to include the DNSimple plugin by adding the import statement immeditely after route53.
+3. Modify the `plugin.cfg` file to include the DNSimple plugin by adding the import statement after route53.
 
 ```txt
 route53:route53
 dnsimple:github.com/dnsimple/coredns-dnsimple
 ...
 ```
 
-4. Load the CoreDNS DNSimple Plugin
-Utilize go get to download the DNSimple plugin, making it available for the upcoming build process.
+4. Get the CoreDNS DNSimple Plugin:
 
 ```shell
 go get github.com/dnsimple/coredns-dnsimple
 ```
 
-5. Build CoreDNS with the External Plugin
-Trigger the build process, generating a new CoreDNS binary that includes the DNSimple plugin.
+5. Build CoreDNS with the DNSimple Plugin:
 
 ```shell
 go generate && go build
 ```
 
-6. Verify and move the binary to a location in your PATH
+6. Verify and move the binary to a location in your PATH.
+
+</details>
+
+<details>
+<summary>B: Download the Precompiled Binary</summary>
+<br>
+
+1. Go to the [DNSimple CoreDNS releases page](https://github.com/dnsimple/coredns-dnsimple/releases)
+2. Download the latest release for your OS
+3. Extract and move the binary to a location in your PATH
+
 </details>
```

Comment:
I think we should also mention the Docker setup as that might be the easiest way to get started for many. This is not a blocker.